### DISCUSSION TITLE
Welcome to HVI Discussions!

### DISCUSSION THREAD

**Discussion by Rootthecause** [Rootthecause/HVI#1] [Announcements]
## Hello! 
Please use the Discussions section here instead of reddit for everything HVI related :) 
Thank you!

**Comment by nnikospapadakis** [comment]
Are there specific components that you suggest using? i mean part numer-wise

**Reply by Rootthecause** [reply]
You can use the parts linked in the BOM, part numbers included :)

For this project the most critical parts are the Inductors and Diodes. 
The inductors should be shielded, in order to reduce EMI to the sourrounding and have high widstand voltage (min. 600V). However the widstand voltage is often not given in the datasheets, thats why I used two in series just for extra precaution. Ideally they should also have low parasitic capacitance, reducing current spikes (look out for highest resonant freqeuncy). 
On the other hand the diodes should be very fast (e.g. ≥ 75 ns), have low capacitance and low forward voltage (min. 1000V breakdown voltage). Althought the US3M is not the best diode, I used it beacuse of its large footprint, resulting in better clearance for single layer design. However, if I recall correctly, I've used a VS-E7FX0212-M3/I for D1 because I ran into issues with the voltage regulation when I used the US3M. 

Edit: I just uploaded the BOM as csv and xml in addition to the ibom. See doc/bom/ 
I also found proof, that I've indeed exchanged D1 with the VS-E7FX0212-M3/I so I've changed the BOM and Footprint as well. 
If you ordered the circuit board before today (December 14, 2024) the Fooprint old will be too large. Please use solder or a short wire to bridge the gap. Using the US3M-13 will lead to issues with the voltage regulation. Sorry for the inconvenience!

**Comment by nnikospapadakis** [comment]
thanks for your help i have been trying to get my design running and yours is a great help, i see that it is important to have wider tracks, in my first design i didnt and i had problrms

**Reply by Rootthecause** [reply]
No problem! :)
Wider tracks do lower parasitic inducance and resistance, but it's not that important in this project. I just prefer utilizing all available space.

**Reply by nnikospapadakis** [reply]
my first design i used a flyback converter andi read that wider tracks it said are important, im a freshman in engineering so excuse me if i dont know much... So in your design wider tracks are not so important? Jlc says that they can do till 0.245mm(notwide)  :( is it a problem if i use these kind of tracks in your design?

**Reply by Rootthecause** [reply]
Just beacuse JLC can do it, does not mean you should 😅
0.245 mm is okay for signals, but maybe not for higher currents. KiCAD has a built in calculator tool for track widths if you know the currents. You could use the LTspice simulation if you want to know it precisely, but thats quite a hassle. 
If you understood the inner workings of a step-down converter, then you can ball park estimate the currents involved, and choose wider tracks accordingly. If you have the space, then you could used wide tracks everywhere. 

I'm here to help with specific questions about the design. But I cannot help you answer basic universal design questions.

**Comment by nnikospapadakis** [comment]
The R6 resistor you have, what should i put in its place?

**Reply by Rootthecause** [reply]
R6 is a do not place part ;) Its just for testing. You can ignore it.

**Reply by nnikospapadakis** [reply]
in the bom you have some parts that instead of a link they have "book" what does this mean?

**Reply by Rootthecause** [reply]
Those are standard parts from a sample book I've got. Any part with the same value will do.

**Comment by nnikospapadakis** [comment]
in your updated pcb design did you changed the d1 from us3m-13 to  VS-E7FX0212-M3?

**Reply by Rootthecause** [reply]
Yes. See [Commit details.](https://github.com/Rootthecause/HVI/commit/957e34bc7acdd70cd4517f527345dd051972af11)

**Comment by nnikospapadakis** [comment]
Hello, i have one question regarding the part JP1 the solder jumper, i cant seem to find any part number related to this, i have looked the web but in vain, can you help me find the right part? is it something you have?

**Reply by Rootthecause** [reply]
It's a solder jumper. You use Solder to bridge two pads together and choose between direct connection or resistor in series for the output.

**Comment by Pierre-Ga** [comment]
Hello, I have one question regarding your components, you use C10 (10uF) and C11 (47uF) on your schematics after PS1, which one did you use in your final design ? (because the value is not the same and i don't know which one is better)
Thank you

**Reply by Pierre-Ga** [reply]
And can I delete R6 an put a cable instead because you said it's just for testing, it's not an actual resistor ?

**Reply by Rootthecause** [reply]
I just used C10. But if you are intend to drive a larger load or do not want to use the small 0603 footprint, then you can use C11 (1210) instead. 

You can delete R6 and leave pin 3 (LIM) unconnected (floating). Do not short R6 !

**Reply by Pierre-Ga** [reply]
Ok, thanks a lot.
And sorry for the stupid question, but I'm new in electronics, but for the fuse that you give in the BOM, does it come with the support suitable for the footprint you put ?

**Reply by Rootthecause** [reply]
Oh, I'm sorry! 
It seems that I forgot to add the fuse clips. It's now added as F1.1 and F1.2 in the BOM. 

Please pull the latest git files. I've added comments to the schematic which have been asked here.
On an unrelated note: I've changed C2 from 1 uF to 100 nF and marked C6 as DNP due to slow regualtion with dynamic loads (I've just discovered this with another project where I used the same schematic as power supply). Please keep the C6 footprint around in case of oscillations or premature overload shutdown.

**Reply by Pierre-Ga** [reply]
Ok thank you

**Reply by Pierre-Ga** [reply]
C6 is like R6, we should not short this one too ?

**Reply by Rootthecause** [reply]
correct. 
I'm a bit concerned about the idea that a component that doesn't need to be there would need to be shorted instead. If it was intended as a short, there would have been a 0Ω Jumper resistor instead. 
It's also a very bad idea of shorting the feeback sensing pin to ground 😉 
I really recommend you to read the datasheet of the viper06.

**Reply by Pierre-Ga** [reply]
Ok I will do that thank you for your time

**Comment by nnikospapadakis** [comment]
hello, i came across one problem, although i have connected everything i get wrong voltage on pin3 (LIM). I have somewhere north of 3.4 where datasheet clearly states that i need to have up til 2.8. Any thoughts of what could have gone wrong?

**Reply by nnikospapadakis** [reply]
also at pin2 i measure 6v if this helps

**Reply by nnikospapadakis** [reply]
i managed to fix some issues and now the led on the back side turns on and off continiusly, should i add a capacitor to c6 empy spot?

**Reply by nnikospapadakis** [reply]
while i measure with my voltmeter the led keeps keep doing on and off, when i dont measure and touch it it stops, and turns on continusly and my dc power supply autolimits it self to 22V

**Reply by Rootthecause** [reply]
Hi! 
I've never measured pin 3 (LIM) as it is not relevant for this converter. I agree, that 3.4 V seems more than the given absolute maximum rating, but I don't how you've measured it. Did you use a multimeter or Oscilloscope? Is anything of your measurement devices connected to ground? You need to understand, that everything directly connected to the IC is oscillating between HV+ and roughly HV-. So if your're going to measure something there, the connected device will also oscillate with the switching frequency of the converter. Your measurement device therefore will act as an antenna and happly radiate EMI. At the same time, it is prone to measurement errors and also induce a capacitive load. So if you're measuring a sensitive pin, you will also influence the chip. 

If you're touching it somewhere near to the IC, your body also acts as an antenna/capacitance - so let me put it like this: just don't touch it :D Especially not when working with HV. 

6V on the VCC's pin would be too low. It should be around 12V. However, the same thing applys with measuring here. Also note, if you're using a voltmeter, then it might not read accurately, as most RMS meters have trouble measuring voltages with overlaying frequencies above 1 kHz or so. But here anything between 500 Hz and 30 kHz is possible (altough the capacitor should dampen it by a lot). 

C6 should not be occupied. However, I found that increasing C2 from 100 nF to 470 nF can improve regulation with large ocillating loads like fans. You could try this, but if you're only using the backside LEDs then it should not make a difference.

For the replica it is much more important, that the voltage across PS1 (pin 1 and 2) is around 12V. What are you measuring there? (It is safe to measure there with a normal voltmeter). Also be sure, that your Input voltage is above 40V otherwise the converter might not start correctly. I recommend for testing, that PS1 is installed only after the correct output voltage is measured.

Regading to your supply: What current limit did you set? If you replica is according to my plans, then it should draw around 4-5 mA. 
if it is higher then that, then there might be an error somewhere. Definetely check the polarity of all diodes (especially D7). 

If you like, you can add pictures of your build. I might be able to identify errors.

**Comment by nnikospapadakis** [comment]
Hi,

For voltage measurements, I primarily use a digital voltmeter, and I connect the ground either to my power supply ground or to the Viper’s GND—I’ve tried both. I also have access to an oscilloscope and can use it for further analysis if needed.

I did touch the IC briefly, as I didn't had the thermal camera on hand and wanted to check for overheating—though, of course, I don’t usually touch or "lick" HV circuits! 😄

Since my initial goal was just to test the LEDs, I didn’t populate a capacitor at C6, at first. At PS1, I measure 12V (actually 13.6V) across pins 1 and 2 (Pin1 = 13.6V, Pin2 = 0V). However, I do not measure the expected 5V on the LV side.

For initial testing, I powered the PCB at 60V DC. However, at some point, the voltage from my power supply suddenly dropped to around 20–30V, depending on the output current limit I set (which I varied from 50mA to <250mA). The power supply then indicated a constant current? (C.C) error, suggesting a short circuit or excessive current draw.

I also observed that when my power supply does not detect an error, the LED is oscillating. However, when the power supply detects an error, the LED remains steady, but at that point, the supply voltage drops to 22V and D7 overheats.

I'll proceed with testing the PCB without PS1 and share my results. Initially, my current limit was set to 50mA, and everything seemed to be working fine. However, after encountering this issue, the PCB started drawing excessive current, causing D7 to overheat and emit smoke, which clearly indicates an issue. I initially considered using a larger diode, but I now suspect that I may have installed it with the wrong orientation. And reverse polarity should for sure cause an issue. (I like how i doubled checked to see if i have correct)

PS: I was about to upload pictures, but I realized that I had modified all reference designators based on your CERN patent. I initially misunderstood the usage rights after reading something about it on GitHub but hadn’t reviewed that section thoroughly. Having said that, I’ll be happy to send pictures. Also, please let me know what I should include on the PCB silkscreen to ensure proper attribution. Also don't mind the qr code its a joke me and my friend have.

Thank you very much for the help, i really hope we find the issue!


https://github.com/user-attachments/assets/3f17ca42-a04b-4619-9a20-058b5a0b6511

![viber_image_2025-02-15_12-12-53-627](https://github.com/user-attachments/assets/f8070339-9f3d-43a9-a6b5-4c191fef3d9a)
![viber_image_2025-02-15_12-12-53-500](https://github.com/user-attachments/assets/4591cc43-c87d-41b2-b328-0909482ed7f7)

**Reply by Rootthecause** [reply]
Hi! 
Thanks for the pictures and video! Thats very helpful to debug the issue!

Haha, thanks for clarification :D 

If D7 is overheating, then the output voltage is too high - which totally makes sense as you've measured 13.6 V on the TS-Side output. Therefore D7 is limiting the voltage as expected. It sounds to me, that there is an issue on the voltage feedback. From the pictures, R7 is labled as "102" which should be 10*10^2 Ω = 1000 Ω. But 10 kΩ is required. If this is the case, the output voltage is set to 92.4 V !
... That would explain A LOT :D 
D7 is meant to protect the output from transients and slight idle over voltage, but not permanent overvoltage. In your case, the exessive over voltage likely destroyed the diode, thus eliminating the protection and therefore PS1 getting much higher voltges. As PS1's output is unregulated, I assume, you've measured higher Voltages than 5V ?

Your To-Do: 
- Please check/replace R7 and D7!
- I would recommend replacing PS1 (13.2V max.) and C8, C9 (16V max.) as those might be faulty.

Regarding the silk screen: 
Don't worry about the reference designators - you can happily change them, as long, as everyone knows what was meant :D (speaking of: I'm referring to the designators from my original, so please check if e.g. R7 is the same or different name in your project). I also don't mind QR-Codes or pictures on the PCB 😄 I originally had pikachu on the pcb, but had to remove it due to copyright.
I'm just caring about, if I was attributed correctly. So in your case something like "Modified by [Your Name] based on a design by Rootthecause" or just "Based on a design by Rootthecause" on the silk screen would be enough. However - as the Cern license (not a patent!) states, many things apply for commercial use. As you/you'r team is likely to be something private, you don't need to follow most of it. So the best I can say: It's not very cool, but it's not against any rules 😉 But I like the fact, that you admitted to this error :) So don't worry.

By the way, I noted that the mounting holes are plated (was originally not intended). I've checked the distances, and belive it should be ok, as long as nylon screws are used.

**Reply by nnikospapadakis** [reply]
I wanted to clarify my understanding regarding the resistor change. The resistor you mentioned (102) is connected to the 5th pin (COMP) and is labeled as R4 on both the PCB file and the interactive BOM.
From my observations, R7 is a 10k resistor connected to the feedback pin. Given this, I just want to confirm whether the change should be applied to R4 or R7. I'll continue testing a few things based on your feedback, but I appreciate any clarification you can provide.
Thanks

**Reply by Rootthecause** [reply]
oh... my bad. You're right! I'm so sorry 🥲
But still, you should not get such overvoltage. 
If R5 and R7 are correct, you could test lowering R5 to 22k, this should result in approx. 10.6 V (+ D1 forward voltage).
Let me know what output voltage you'll get then.

**Reply by nnikospapadakis** [reply]
That worked out great. Thank you very much. I still have to do the required changes on the silkscreen and solder again the components for better reliability. If i have any further questions ill ask. I want now to test with a led with cables.

**Reply by Rootthecause** [reply]
So 22k for R5 was the fix? Please let me know what the Voltage across Pin 1 and 2 of PS1 is. This would be very helpful, as others might have this problem as well.

**Reply by nnikospapadakis** [reply]
So, between pin1 and pin2 of PS1 the voltage is 13.01V (still a bit high in my understanding, also my R5 is exactly 21.3KΩ). And on the LV side pin 3 and 4 is 5.5V, note that i havent changed the PS1 for a new one after all the trial and error

**Reply by Rootthecause** [reply]
Indeed, thats still a bit high. How does the Voltage change when load is applied? 
5.5V on the output of PS1 seems fine to me. They seem to have slightly higher output voltage in general, so that nominal voltage is reached under load. If PS1 is not sigificantly warm, then I suspect it is working fine - no replacement needed.

**Reply by nnikospapadakis** [reply]
What do you mean by load? More than the leds mounted on the pcb? haven't tried it yet..

**Reply by Rootthecause** [reply]
Yeah, I mean e.g. 100 Ohms at the 5V output. But if you have the LEDs connected, then this should be enough load already. I just looked up my test data: When there is no load connected at all (not even the LED D3), then the output voltage can rise around 0.8 V above the set voltage (thats what D7 was originally intended for). But even 3 mA load are enough to get near to nominal voltage. So in your case, I would recommend going even lower with R5, for example 18 kΩ, hopefully resulting in around 12V. My converter resulted in 12,1 V to 12,3 V depending on load and Input voltage. It's weird to see the output voltage that high.

**Comment by Pi3r-2** [comment]
Hello, I want to use your design and I have a question regarding the FSG rules.
I saw in your files that it passed the FSG scrutineering, but when I looked to your components I saw that in the TS side of the PCB you have a component that is not rated for the max TS voltage (like R4, R5, R7, C3...). How does it pass the scrut with EV 4.1.2 "All components in the TS must be rated for the maximum TS voltage."
I'm sorry my question can seem strange, but I don't see why.
Thank you for your time

**Reply by Rootthecause** [reply]
Hi,
it truly says "all components", but this is pretty much impossible to accomplish. Just think of any BMS. If the same logic was applied, even the IC would need to handle 600 V (very unlikely). I'm also certain, that most precharge and discharge circuits used in FS would be not according to EV 4.1.2 (and also any Inverter). I believe it was meant, that A) any component directly between/at the TS potentials in the main current path must be rated for max. TS Voltage, e.g. Fuses, Relais, MOSFETs etc.  and/or B) "components" does not mean something small like a single resistor, but either the whole module like Inverter, Precharge, Discharge.
The intent of the rule probably was, to ensure, that teams use proper voltage ratings e.g. cables which have a rating of 250V but a max. Isolation Voltage of 1 kV, so it handles 600 V as well, right? 🤡 - I can see, that something like this would happen / has happened. 
I just spoke to a different Team's Member who is also working with their HV system. It truly seems, that Scrutineers only care about components, which are relevant to the competition and direcly see TS Voltage.
I will keep this question in mind for further opinions on it, as I belive the wording is quite misunderstandable and might need a change.

Regarding to the HVI, the components you've named have just a couple of volts applied and are not in the main current path of the converter. Even in an event of failure, those component are unlikely to see max. TS Voltage.

**Comment by Giblet2501** [comment]
Dear Roothecause, this hvi can work on 24V, if not, which components should we consider for the schematich?

**Reply by Rootthecause** [reply]
I think thats not possible with this chip, as the Drain-Source startup Voltage is between 25V and 45V. 
If you just use 24V as source, then you can use any normal step-down converter, as the HVI is specifically designed to handle high voltages - but 24V is rather low voltage. Or do you ask for 24V output voltage and not input?

**Comment by enmanuelhdzzz** [comment]
Dear Tooththecause, With the new FS-Rules 2026 V1.1, I believe your current design may no longer comply due to the following clause:

EV5.4.8: Each TSAC must have a prominent indicator, a voltmeter, or a red LED visible even in bright sunlight. If a red LED is used as an indicator, it must continuously illuminate at its full brightness whenever a voltage greater than 60 VDC or half the maximum TS voltage (whichever is lower) is present at the vehicle side of the AIRs.

As things stand, I don’t see a logical output stage (e.g. using a MOSFET or transistor) in your design that forces the LED to full brightness under those conditions. I would recommend adding such a transistor or MOSFET stage driven by a 12 V logic signal (or equivalent) so that the LED is guaranteed to operate at full brightness when required, thus satisfying the rule.

If you already have something equivalent in your design, maybe you could clarify how it ensures the continuous full-brightness requirement under all operating conditions.

**Reply by Rootthecause** [reply]
Hi! 
I've checked the rules yesterday and saw the change in this regard. 
But there is no design change required, as the currently used high-voltage step down is regulating to a constant 5 V output regardless of input voltage, thus it stays at full brightness all the time. 
The only edge case where the LED would not light at full brightness is below approx. 12 V input. However, the rules state, that it must illuminate at "at its full brightness whenever a voltage **greater than 60 VDC**", and this is sufficiently met ;) 

I belive (speculation!) this rule change was made, because some teams used just LEDs with a large resistor in series. So it does light up at 60V but not at full brightness and will reach full brightness only at max. voltage. 

Edit: I've updated the Q&A section on the main page regarding the new rules for better clarification.

**Reply by enmanuelhdzzz** [reply]
perfect you are the best!!

**Comment by BatteryLicker092** [comment]
Hello Roothecause!

I have a question regarding R4 and C7 on the COMP pin. I understand what the purpose is of the RC circuit but I am confused as to how you have decided on 1k and 22nF?

Thank you!

**Reply by Rootthecause** [reply]
Iirc I took them from the evaluation board: https://www.st.com/resource/en/data_brief/steval-isa114v1.pdf

I also did a lot of simulations in LTspice, but I don't certainly remember if/what I've tested in simulation for R4 and C7. It just happened to work with those values. It was way harder to find good values for C6 and C2 as in some load cases (e.g. connecting a 12V 150mA fan) caused an unstable output voltage. You can see the adjustment in this push: https://github.com/Rootthecause/HVI/commit/3586ca1aab562fbaf24d0bd53b2974b4139a8919

**Reply by Rootthecause** [reply]
I just looked up another project I used the viper06 circuit (this was the one where I had the fan driving issues). It seems like I did not change then values for the HVI according to my findings. Maybe I did not had the time to do change that back then, but I can tell those changes where proven working for at least 30 hours. 

<img width="1577" height="671" alt="grafik" src="https://github.com/user-attachments/assets/f3b1691d-781e-4b6d-8ea8-5047b0c114ef" />

This was my written log: 

Fan Driving Issue 241219
At higher input voltage, it is no longer possible to drive a fan (80mm BeQuite).
Overload restart occurs after approx. 1 sec.
Changing D9 from a US3D-13 to a VS-E7FX0212-M3 helped to get stable output voltages but not the fan issue.
Changing D12 from a US3D-13 to a VS-E7FX0212-M3 did nothing

Without changes: max. 60V
With 470 uF across C30 -> max. 200V 
With 2700 uF across C30 -> max. 200V but startup issues
Adding C26 did nothing.
Increasing voltage to 13,1 V (30k/10k) increased it to 65V (even with 470 uf?!)
Without C29 -> overheating due to output overvoltage / bad regulation
Without C27 -> seems to be very fine together with 470 uF up to 630V but without 470uF max. 60V
Without C27 but C29 = 470nF -> max. 600V but 12.7V at output. Across C29 = 13.1 V
Without C27 but C29 = 100nF -> max. 600V but 12.8V at output. Across C29 = 13.1 V
Without C27 but C29 = 33nF -> max. 600V but 12,86V at output. Across C29 = 13.1 V
Without C27 but C29 = 33nF and 220 uF at output -> max. 600V but 12,86V at output. Across C29 = 13.1 V
Without C27 but C29 = 33nF and 47 uF at output -> max. 600V but 12,86V at output. Across C29 = 13.1 V but HF whining
C28 to 1 nF instead 22 nF -> more noise
C28 = 22nF and R29 to 4k7 instead 1 k -> less noise ouptut 1060 mVpp @ 600Vin
C28 = 22nF and R29 to 10k instead 1 k -> less noise -> ouptut 900 mVpp @ 600Vin
C28 = 22nF and R29 to open instead 1 k -> max. 150Vin
C28 = 22nF and R29 to short instead 1 k -> max. 600Vin -> ouptut 1300 mVpp @ 600Vin
C28 = 22nF and R29 to 27k instead 1 k -> max. 151 Vin
C29 = 10 nF, C28 = 10nF and R29 to 10k -> ouptut 900 mVpp @ 600Vin
C29 = open, C28 = 10nF and R29 to 10k -> ouptut 1000 mVpp @ 600Vin
C29 = 100nF, C28 = 10nF and R29 to 10k -> ouptut 1000 mVpp @ 600Vin somewhat slow overregulation
C29 = 100nF, C28 = 10nF and R29 to 1k -> ouptut 1410 mVpp @ 600Vin somewhat slow overregulation
mit 330 uF electrolyte at output -> ouptut 240 mVpp @ 600Vin. pretty good, bt still somewhat noisy.
mit 2700 uF electrolyte at output -> too much for startup
C29 = 100nF, C28 = 10nF and R29 to 4k7-> overload at 50V
C29 = 10nF, C28 = 10nF and R29 = 10k -> 330 mVpp and more audible noise 
C29 = 200nF, C28 = 10nF and R29 = 10k -> 240 mVpp and more quite
C29 = 470nF, C28 = 10nF and R29 = 10k -> 240 mVpp similar to 200nF. Leave it as it is for now.
-> New values transferred to circuit diagram 241219


Edit: fixed remaining german words

**Reply by BatteryLicker092** [reply]
Hello! Thank you for your detailed reply and apologies for my late reply. We have since designed and tested the viper06 circuitry and are running into a few issues. The VDD pin does not appear to be outputting a value within the expected range. We have tried at 55 volts and 40 Volts on the drain pins to see if there is any correlation. We are wondering if you had ran into any issues like this?
<img width="607" height="61" alt="VIPER06" src="https://github.com/user-attachments/assets/5f39594d-be3a-4d23-ba86-811f8b10cc2a" />
<img width="1256" height="587" alt="VIPER06XS" src="https://github.com/user-attachments/assets/5b281c84-0283-421e-827f-2f4aa71e29ef" />

**Reply by Rootthecause** [reply]
Welcome back ;)

I've checked your D1 (SM4007PL)
<img width="819" height="84" alt="Bildschirmfoto 2026-03-15 um 00 39 13" src="https://github.com/user-attachments/assets/a9df335a-2114-495b-bb6d-c1886aef277d" />
The recovery time is way too high! 
I've deliberately choosen a fast recovery diode (VS-E7FX0212-M3/I) with trr = 45 ns in my schematic, as a too long recovery time discharges your C2. Even the US3M-13 is a better option (trr = 70ns), but in my tests it was not good enough, hence the VS-E7FX0212-M3/I. 
Meanwhile I think you could replace D2 and D5 as well with the VS-E7FX0212-M3/I, but I've not done that on my HVI release, as the US3M-13 has a larger footprint, allowing better single sided routing (allows a trace crossing the diode below). 

Anyways, your measurement are between TS- and the stated potential, right?
So the voltage between VDD and GND is just around 0.5 V. 
How did you measure the voltage? Unless you've used a scope, your measurements might be very off from what happens there, as the viper functions like a highside switch. Topology wise, this is just a buck converter with a self oscillating high-side switch and some bootstrapping to supply it. So this means, that anything on the GND potetial of the chip might be either at TS+ or VP_out potential, depending on the switch state of the viper. When is switches properly, the ON-State is just a few hundred ns. 
Using a normal multimeter probably won't pick that up. 
However, the VP_out to TS- potential is smooth out, so the 0.57 V are probably correct. 
Could you please show me a picture of your scope connected to D5 ? There should be some switching apparent. If not, it may be broken or not properly soldered/connected. Did you use your own layout or the one I released?

Btw. which L1 are you using? I do not recommend using a single SMD inductor due to the required voltage rating. But I've seen some rated THT inductors.

**Reply by BatteryLicker092** [reply]
Hey! 
We have replaced D1 for 1N4148WS-7-F as it is what we have in stock and footprint is similar. It has a recovery time of 4ns, very quick. The behaviour of the board remains the same. Do you think that any components could have been fried by having the old diode? 
Many thanks in advance.

**Reply by Rootthecause** [reply]
Caution with this diode. It's only rated for 75V continous. This diode must handle the max. TS voltage, otherwise it probably kills a few components. But for testing at low voltages it should be ok :) 
If it still does not work, then I assume some other components might be broken. Just test them, or replace them alltogether.

**Comment by mencue28** [comment]
How did you calculate the values? Here https://www.st.com/resource/en/data_brief/steval-isa114v1.pdf it appears the values for a converter with an input voltage that is not which I want.
